chapters/03_queueing_model/05_numeric_results_and_timings/main.tex

```diff
@@ -0,0 +1,331 @@
+
+Figure~\ref{fig:comparison_steady_state_probabilities_1} has the smallest
+value of \(\mu = 0.03\).
+It can be seen that for both the Markov chain and the simulation the steady
+state probabilities are close to zero for most states apart from the
+states close to when the system is full.
+That is because the arrival rate of individuals is much lower than the service
+rate of individuals even with 5 servers.
+Note here that because the model is immediately flooded from the beginning, the
+simulation has no time to explore the state space, so the heatmap looks like
+it is missing some of its pieces.
+Additionally, it is interesting to note that as we increase the value of \(\mu\)
+in Figures~\ref{fig:comparison_steady_state_probabilities_2} -
+\ref{fig:comparison_steady_state_probabilities_5} smaller states in the model
+have a higher value since individuals now exit the system faster.
+Also, note that for all values of \(\mu\) the difference between the Markov
+chain approach and the simulation is small.
+
+
+\subsubsection{Simulation VS Markov chain durations}
+
+The choice of the artificial parameters \(N\) and \(M\) is an important 
+decision of the model.
+In the untruncated simulation these values can be omitted by setting them to
+infinity.
+This is not possible when obtaining the steady state probabilities of the
+finite state Markov chain.
+The value of \(N\) and \(M\) can be chosen arbitrarily large, but the
+computation time increases as the size of the state space increases.
+Table \ref{tab:truncation_effect_timings_old} shows the relative timings of the
+different approaches used to get the performance measures for different values
+of \(N\) and \(M\).
+Note that \(N\) and \(M\) have the same value throughout the table.
+Simulation is ran for a runtime of \(10^4\) time units and the displayed
+durations are for a single run of the simulation and similarly for 100 runs of
+the simulation.
+For getting the performance measures using the finite state Markov chain each
+timing recorded is for the computation of the steady state probabilities and
+then the corresponding performance measure.
+
+\tiny
+\begin{table}[H]
+    \centering
+    \begin{tabular}{c|cc|ccc}
+        & \multicolumn{2}{c}{\textbf{Simulation}} & 
+        \multicolumn{3}{c}{\textbf{Markov chain}} \\
+        \textbf{Value of} & \textbf{Single} & \textbf{100} & 
+        \textbf{Waiting} & \textbf{Blocking} & 
+        \textbf{Proportion} \\
+        \textbf{\textit{N} and \textit{M}} & \textbf{trial} & \textbf{trials} & 
+        \textbf{formula} & \textbf{formula} & \textbf{formula} \\
+        \hline
+        \(10\) & 1 & 144.3 & 0.015  & 0.014  & 0.014 \\
+        \hline
+        \(30\) & 1 & 143.4 & 3.73   & 3.83   & 3.65 \\
+        \hline
+        \(50\) & 1 & 139.8 & 31.57  & 38.39  & 31.98 \\
+        \hline
+        \(\infty\) & 1 & 142.1 & N/A & N/A & N/A \\
+    \end{tabular}
+    \caption{Relative timings of the simulation and Markov chain model}
```

Comment:
Move the caption to above the table. I don't know why, apparently captions for tables are supposed to be above tables.